lib/happo/public/HappoDiffs.jsx

```diff
@@ -3,24 +3,32 @@ const PropTypes = React.PropTypes;
 const imageObjectStructure = {
   description: PropTypes.string.isRequired,
   viewport: PropTypes.string.isRequired,
-  url: PropTypes.string.isRequired,
+  diff: PropTypes.string.isRequired,
+  previous: PropTypes.string,
+  current: PropTypes.string,
 };
 
 function imageSlug(image) {
   return btoa(image.description +  image.viewport);
 }
 
-function renderImage(image) {
+function renderImageHeading(image) {
```

Comment:
unrelated: These are very close to being stateless functional components. Might as well go all the way.